package io.medium.poc.common.exception

class CallProceduresException(message: String? = "프로시져 호출 중 에러가 발생했습니다.") : RuntimeException(message)